reflection/reflection.fbs

```diff
@@ -29,6 +30,7 @@ enum BaseType : byte {
 table Type {
     base_type:BaseType;
     element:BaseType = None;  // Only if base_type == Vector.
+    fixed_length:short = 0;   // Only if base_type == Array.
```

Comment:
same here, this is an incompatible change. has to be at the end, or use ids. or maybe reuse `index` for this purpose.